content/blog/2021/046-getting-erc20-transfer-events.md

```diff
@@ -194,11 +194,11 @@ Additionally, understand that the TrueBlocks API mirrors exactly the command lin
 For example, the above command-line query could be re-written as a curl command thus
 
 ```[bash]
-curl "http://localhost:8080/export?addrs=0x03fdcadc09559262f40f5ea61c720278264eb1da&logs&articulate&fmt=csv&max_records=10000" \
+curl "http://localhost:8080/export?addrs=0x03fdcadc09559262f40f5ea61c720278264eb1da&logs&articulate&fmt=csv&maxRecords=10000" \

```

Comment:
Excellent catch. You inspired me to look and I found another instance of the same problem in this file: ./blog/2021/048-listing-tokens-i-own.md.

Can you change that file too?

Reply:
Changed :sunglasses: